contracts/extensions/StakedExpenditure.sol

```diff
@@ -109,14 +110,16 @@ contract StakedExpenditure is ColonyExtensionMeta {
   /// @param _value Reputation value indicating the total reputation in _domainId
   /// @param _branchMask The branchmask of the proof
   /// @param _siblings The siblings of the proof
+  /// @param _stagedPayment Whether the expenditure is a staged payment
   function makeExpenditureWithStake(
     uint256 _permissionDomainId,
     uint256 _childSkillIndex,
     uint256 _domainId,
     bytes memory _key,
     bytes memory _value,
     uint256 _branchMask,
-    bytes32[] memory _siblings
+    bytes32[] memory _siblings,
+    bool _stagedPayment
```

Comment:
To clarify, the idea here is that you can toggle between this being a staged payment or not.

I'm assuming this has been included because it was desired that you could add or remove staged payments while in draft and removing some so that there is only one would result in it not being needed as a staged payment?

Otherwise, no, being able to change an expenditure type from say split to staged or vice versa while it draft does not need to be possible.

Reply:
> Otherwise, no, being able to change an expenditure type from say split to staged or vice versa while it draft does not need to be possible.

Grand, then the current functionality is okay.